packages/cli/src/commands/build.ts

```diff
@@ -538,8 +561,8 @@ async function doBuild(
 
   // Merge existing `config.json` file into the one that will be produced
   const configPath = join(outputDir, 'config.json');
-  // TODO: properly type
```

Comment:
I resolved this TODO because I was working with this anyway.

The type `BuildOutputConfig` is duplicated in the api repo, though. Should we move this to a central place, like `built-utils`?

Reply:
`build-utils` would be fine I think.